assets/css/main.scss

```diff
@@ -2,51 +2,171 @@
+            border-color: $brush-white;
+        }
+
+        .navbar-header{
+            .navbar-brand {
+                color: $brush-white;
+            }
+        }
+
+        .navbar-collapse {
+            >ul.nav {
+                >li {
+                    >a {
+                        color: $brush-white;
+                        &:hover {
+                            opacity: 0.9;
+                            color: $brush-white;
+                        }
+                    }
+                    &.dropdown {
+                        &.open {
+                            >a {
+                                color: $brush-white;
+                                background-color: lighten($brush-accent, 5%);
+                            }
+                        }
+
+                        
+                        >ul.dropdown-menu {
+                            border-radius: 0;
+                            background-color: $brush-accent;
+
+                            >li {
+                                >a {
+                                    color: $brush-white;
+                                    &:hover {
+                                        opacity: 0.9;
+                                        color: $brush-white;
+                                        background-color: lighten($brush-accent, 5%);
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    > .container {
+        padding: 60px 15px 0;
+        .text-muted {
+            margin: 20px 0;
+        }
+
+        .embed-wrapper {
+            display: flex;
+            width: 100%;
+            margin-bottom: 5rem;
+
+            >figure {
+                margin-right: 0.5rem;
+                margin-bottom: 0;
+                order: 1;
+                flex: 1;
+            }
+            >.frame-container {
+                margin-left: 0.5rem;
+                order: 1;
+                flex: 1;
+                >header {
+                    width: 100%;
+                    display: flex;
+                    justify-content: space-between;
+                    >span {
+
+                    }
+                    >a {
+                        margin-right: 0;
+                    }
+                }
+                >iframe {
+                    width: 100%;
+                }
+            }
+        }
+
+        >div.language-html {
+            margin-bottom: 5rem;
+        }
+        
+    }
+
+    code {
+        font-size: 80%;
+    }
+
 }
 
-#footer {
-  background: #333;
-  color: #DDD;
-  margin: 2em 0 0;
-  padding:.5em 0 2em;
-
-  h2 {
-    font-size:1.2em;
-  }
-  a {
-    color: #AAA;
-    text-decoration:none;
-  }
-  a[href] {
-    color: #FFF;
-    text-decoration:underline;
-  }
-  ul {
-    padding-left:1em;
-  }
-}
\ No newline at end of file
+
+// table {
```

Comment:
Either this shouldn't be commented out, or this shouldn't be here at all :)

Reply:
Removed commented styles.